### ISSUE TITLE
table_mode_separator

### ISSUE THREAD

**Issue by vasyapupkin123456** [dhruvasagar/vim-table-mode#77]
let g:table_mode_separator = '┊'
E121: Undefined variable: separator_map
E15: Invalid expression: separator_map . '<Esc>:call tablemode#TableizeInsertMode()<CR>a'

**Comment by dhruvasagar** [comment]
How would you type that character ? The separator is also used to bind they key for table creation.

**Comment by vasyapupkin123456** [comment]
in insert mode C-k 4! --> ┊

**Comment by dhruvasagar** [comment]
wouldn't it be very awkward to type ? Table Mode uses the separator as a mapping in insert mode to add columns / rows / borders. Or would you instead use the separator as that but still use `|` for mapping ?

**Comment by vasyapupkin123456** [comment]
I make programming notes in vim. And many notes have table:
http://postimg.org/image/do34bu61j/

imho, one way do this - used separator not from writable symbols

**Comment by dhruvasagar** [comment]
Actually that is a known problem / issue. I still haven't found a very good solution to it yet. I am working on it. You're suggestion is worth considering. Thanks. I will notify via this issue once I have resolved it.

**Comment by vasyapupkin123456** [comment]
If use hotkey not from writable symbol, It would be good to make a hotkey to call the separator

**Comment by qlyoung** [comment]
I agree that while | is very convenient for initial use, I also use this plugin to document grammars and the vertical bar is the canonical symbol for `or` in many programming languages. Would like an unprintable binding if possible.
